ios-shim/src/lib.rs

```diff
@@ -0,0 +1,60 @@
+use core_foundation::base::{CFRetain, OSStatus, TCFType};
+use core_foundation::data::{CFData, CFDataRef};
+use core_foundation::string::{CFString, CFStringRef};
+use security_framework_sys::base::{errSecBadReq, errSecSuccess};
+
+#[no_mangle]
+pub extern "C" fn SecSetGenericPassword(
```

Comment:
This uses Apple's `Sec*` naming scheme. All symbols in C are global, so there's a risk of a collision with Apple's functions. If this function is just for Rust, please add a custom name prefix, e.g. `rust_sec_framework_set_generic_password`

Reply:
This function is actually just for use from Swift and Objective C/C++ code, both of which use camel case naming.  I am happy not to use Apple's naming convention, but would prefer to keep using camel case.  How about `Rust` or `RustSec` or `RustShim` as a prefix, as in `RustShimSetGenericPassword`?  Do you have a preference?

Reply:
RustShim is fine.

Reply:
OK, will use `RustShim`